editor/inspector/multi_node_edit.cpp

```diff
@@ -148,18 +148,20 @@ void MultiNodeEdit::_get_property_list(List<PropertyInfo> *p_list) const {
 				F.name = F.name.replace_first("metadata/", "Metadata/"); // Trick to not get actual metadata edited from MultiNodeEdit.
 			}
 
-			if (!usage.has(F.name)) {
+			PLData *usage_data = usage.getptr(F.name);
+			if (!usage_data) {
 				PLData pld;
 				pld.uses = 0;
 				pld.info = F;
 				pld.info.name = F.name;
 				usage[F.name] = pld;
-				data_list.push_back(usage.getptr(F.name));
+				usage_data = usage.getptr(F.name);
```

Comment:
You can replace it with
```suggestion
				HashMap<String, MultiNodeEdit::PLData>::Iterator I = usage.insert(F.name, pld);
				usage_data = &I->value;
```
although not sure if getting the pointer is safe here.

Reply:
Looks like the `Iterator` just stores a `HashMapElement *` and `I->` is actually an operator overload for `I.data->`, so the pointer will end up pointing to a property of a persistent `HashMapElement` and should be safe to use.